quipucords/api/source/model.py

```diff
@@ -52,6 +52,22 @@ class Source(models.Model):
         "api.ScanJob", null=True, on_delete=models.SET_NULL, related_name="+"
     )
 
+    @property
+    def options(self):
+        """Return the v1 compatible options attribute for the source object."""
+        result_options = {}
+        if self.ssl_protocol is not None:
+            result_options["ssl_protocol"] = self.ssl_protocol
+        if self.ssl_cert_verify is not None:
+            result_options["ssl_cert_verify"] = self.ssl_cert_verify
+        if self.disable_ssl is not None:
+            result_options["disable_ssl"] = self.disable_ssl
+        if self.use_paramiko is not None:
+            result_options["use_paramiko"] = self.use_paramiko
+        # if result_options == {}:
+        #    return None
```

Comment:
I think we can get rid of this comment, right?
(this reminded me, I'm almost sure there's a checker on ruff for commented code)

Reply:
ah, nevermind. it was removed in a subsequent commit.